src/OrchardCore.Modules/OrchardCore.Flows/Views/FlowPart.Edit.cshtml

```diff
@@ -15,15 +16,40 @@
 <script asp-name="jQuery-ui" at="Foot"></script>
 
 <fieldset class="form-group">
-    <div id="@widgetTemplatePlaceholderId" class="widget-template-placeholder">
+    <div id="@widgetTemplatePlaceholderId" class="widget-template-placeholder row " style="min-height:50px;">
         @{ var htmlFieldPrefix = this.ViewContext.ViewData.TemplateInfo.HtmlFieldPrefix; }
         @foreach (var widget in Model.FlowPart.Widgets)
         {
             var prefix = Guid.NewGuid().ToString("n");
-            <div class="widget-template">
-                @{
-                    dynamic widgetEditor = await ContentItemDisplayManager.BuildEditorAsync(widget, Model.Updater, false, "", prefix);
+            dynamic widgetEditor = await ContentItemDisplayManager.BuildEditorAsync(widget, Model.Updater, false, "", prefix);
+
+            var metadata = ((ContentItem)widgetEditor.ContentItem).As<OrchardCore.Flows.Models.FlowMetadata>();
 
+            var colSize = 100;
```

Comment:
Could we move this into function?

Reply:
colSize / 12 = percentile so ... 

var colsize = Math.Round(percentile * 12);